info-compiler/src/test/java/com/google/sps/NewsContentExtractorTest.java

```diff
@@ -41,57 +46,83 @@
   private final static String WRONG_URL =
     "https://www.wrong.com";
   private final static String EMPTY = "";
-  private final static NewsArticle NEWS_ARTICLE =
-      new NewsArticle(
-          "AOC wins NY Democratic primary against Michelle Caruso-Cabrera, CNN projects - " +
-          "CNNPolitics",
-          URL,
-          "Washington (CNN)Freshman Democratic Rep. Alexandria Ocasio-Cortez will defeat former " +
-          "longtime CNBC correspondent and anchor Michelle Caruso-Cabrera in a Democratic " +
-          "primary election on Tuesday for New York's 14th Congressional District, CNN projects.");
-  private final static NewsArticle NEWS_ARTICLE_WITH_WRONG_URL =
-      new NewsArticle(
-          "AOC wins NY Democratic primary against Michelle Caruso-Cabrera, CNN projects - " +
-          "CNNPolitics",
-          WRONG_URL,
-          "Washington (CNN)Freshman Democratic Rep. Alexandria Ocasio-Cortez will defeat former " +
-          "longtime CNBC correspondent and anchor Michelle Caruso-Cabrera in a Democratic " +
-          "primary election on Tuesday for New York's 14th Congressional District, CNN projects.");
-  private final static NewsArticle NEWS_ARTICLE_WITH_EMPTY_CONTENT =
+  private final static String TITLE = "News Article Title";
+  private final static String CONTENT = "News article content.";
+  private final static TextBlock TEXT_BLOCK = new TextBlock(CONTENT);
+  private final static TextDocument TEXT_DOC_REGULAR =
+      new TextDocument(TITLE, Arrays.asList(TEXT_BLOCK));
+  private final static TextDocument TEXT_DOC_WITHOUT_TITLE =
+      new TextDocument(Arrays.asList(TEXT_BLOCK));
+  private final static NewsArticle NEWS_ARTICLE_REGULAR =
+      new NewsArticle(TITLE, URL, CONTENT);
+  private final static NewsArticle NEWS_ARTICLE_REGULAR_WRONG_URL =
+      new NewsArticle(TITLE, WRONG_URL, CONTENT);
+  private final static NewsArticle NEWS_ARTICLE_WITHOUT_TITLE =
+      new NewsArticle(EMPTY, URL, CONTENT);
+  private final static NewsArticle NEWS_ARTICLE_WITH_EMPTY_TITLE_CONTENT =
       new NewsArticle(EMPTY, URL, EMPTY);
-  private final static int END_OF_STREAM_INDICATOR = -1;
-  private InputStream realWebpageStream;
+  private InputStream webpageStream;
   private NewsContentExtractor realNewsContentExtractor = new NewsContentExtractor();
 
   @Before
   public void createInputStream() throws IOException {
-    realWebpageStream = new URL(URL).openStream();
+    webpageStream = new URL(URL).openStream();
```

Comment:
Do you need this to be a real inputstream? Probably can just have a mock object instead of opening a sream.